Update ppb defund (#1538)
defund12/defund12.org: _emails/us/oregon/portland_defund_ppb.md

- sonia.schmanski@portlandoregon.gov
- Toni.Anderson@portlandoregon.gov
- Andrew.Bryans@portlandoregon.gov

> body: "Hello,
> 
> My name is [YOUR NAME] and I am a resident of Portland. I would like to see us as a city committed to disbanding the Portland Police Bureau (PPB) immediately. We must get over this hurdle so that we can start to make real change.

Reviewer: ```suggestion
body:  |-
Hello,

My name is [YOUR NAME] and I am a resident of Portland. I would like to see us as a city committed to disbanding the Portland Police Bureau (PPB) immediately. We must get over this hurdle so that we can start to make real change.
```

I am happy to see the proposed cuts in over $15 million to PPB championed by Commissioner Hardesty, and at this time I need to see more. I see the collective call for $50 million in reductions and believe this is insufficient- I want to see an immediate $100 million reduction to the PPB in the 20-21 FY budget cycle. Now is the time to take decisive action, not next year or the next, when public scrutiny will perhaps be less intense. Set a course for the full disbandment of the Portland Police Bureau NOW, or I fear it will never happen.

I am skeptical of Wheeler's ability to steer us clearly as city mayor with his position at the helm of the PPB as Commissioner. The truth is, we need cuts and a commitment to disband, like the cities we see before us setting the stage for such change at this moment in time.

We need to know that we are being taken seriously and that we mean for the PPB to be dismantled in support of building Black futures and community care. The PPB does not protect us equally and we can do better.

> Specifically, in accordance with the demands of Care Not Cops and the design to defund and reallocate, I ask for:

Reviewer: ```suggestion
Specifically, in accordance with the design to defund and reallocate, I ask for:
```

Reviewer: refrain from endorsing outside organizations

Author: My understanding is that letters should refrain from endorsing outside orgs without permission of said org. Is this a general rule (which I see the logic of), or would it be ok to keep this given specific written permission from Care Not Cops?

Reviewer: It is a general rule

1) Immediate elimination of all vacant positions within the PPB, and to fold this funding back into the general and special appropriations funds.\n\n2) Implementation of an immediate hiring freeze, and a commitment to ending any new hires of Portland Police Bureau sworn officers.

3) Immediately eliminate all funds for the following specialty units (those already included in the proposed item are not listed here): Neighborhood Response, Service Coordination Team, Behavioral Health Unit, Air Support Unit and Rapid Response Team (Riot Control)
  3.a) Redirection of these funds towards anti-violence programs unaffiliated with law enforcement, reparations for Black Portlanders targeted by gang policing, and other investment in Black people to be determined by impacted communities and grassroots groups such as Don’t Shoot Portland & Freedom to Thrive.
  3.b) Eliminate these positions entirely by reducing the total number of officers, not just by folding them back into general sworn officer positions.

> 4) Immediately stop policing protests - officers are bringing the riot, not the demonstrators. Even if they stop using tear gas, glass beads, plastic and rubber bullets, and other 'less lethal weapons' they will continue to behave how they know and shift to other ways to enact violence.

Reviewer: This is nice, but will have to be amended when pr
```suggestion

```

A great point for right now, but would have to be updated later, so I suggest removing until we have time senstive letters.

Author: Great point, thank you!

5) End PPB relationship to the Federal 1033 program, which supplies militarized weapons, gear, and vehicles to local police departments across the country.

6) Divest City of Portland Cannabis Tax funds from the PPB budget AND invest them instead into reparations for those harmed by the War on Drugs, which disproportionately targeted Black communities for activities that are now legal.

> 7) Immediately stop expansion of the Portland Police Bureau budget by ending all budget increases. The City Council must commit to never vote for PPB Budget increases again.

Reviewer: move this point closer to top

8) Immediately end the use of police responders to health or mental health calls and wellness checks, and implement alternative responses that are not led by or contingent on interaction with the police.

9) Immediately terminate contract between the Portland Police Bureau and Portland Business Alliance for the Clean & Safe Enhanced Services District.

10) Immediately eliminate police presence in historically Black neighborhoods, communities of color, and areas with high houseless populations in order to reduce the fear, ongoing trauma, violence, and oppressive conditions which impede people’s daily survival.

> Finally, I need to see that there will be allocations to the bereaved families of the citizens of Portland killed by members of the PPB. I urge that the testimonies of those who have come forward be reviewed, as the names of the officers at fault and the victims that have fallen at their hands have been adequately named.

Reviewer: not completely on message

The PPB has been caught in collusion with prominent white nationalist groups, shootings of Black people and people perceived to be in mental health crisis, and this violence and terrorism against our citizens needs to stop NOW.

> I call on the Mayor, the Commissioners, and all other City Council Members to use their powers to continue to further reduce the budget and pledge to increase Police accountability and transparency. I promise that you will lose my support and my vote if you aren’t willing to make a public and vocal stand to truly defund the PPB, increase their accountability and transparency, and do due diligence to ensure that we no longer need to use the hashtag Black Lives Matter because it will be evident in our practices.

Reviewer: ```suggestion
I call on the Mayor, the Commissioners, and all other City Council Members to use their powers to continue to further reduce the budget and pledge to increase Police accountability and transparency. I promise that you will lose my support and my vote if you aren’t willing to make a public and vocal stand to truly defund the PPB, increase their accountability and transparency, and do due diligence to ensure that we nobody needs to use the Black Lives Matter hashtag anymore because it will be evident in our practices.
```
